### ISSUE TITLE
Art Search Limit setting ovewritten on write new config

### ISSUE THREAD

**Issue by stebe** [ampache/ampache#2989]
### Describe the bug

Whenever I write a new config the setting for art_search_limit gets reset to defaults.

### To reproduce

1.  Remove comment for art_search_limit in ampache.cfg.php
2. Set a value and save the config
3. From web interface go to ampache debug
4. Write new config

Diff of written config

```
diff ampache.cfg.php-2021-08-12 ampache.cfg.php
655c655
< art_search_limit="25"
---
> ;art_search_limit="15"

```

## Environment

* Ampache version: 107a1b4c75942c4841dc3d62ede2eba9e0e8c5b0

### Client type

* Web

**Comment by lachlan-00** [comment]
hmm, good find maybe  theres a list of config options somewhere to verify, i'll check now

**Comment by lachlan-00** [comment]
need to work on the regex, it doesn't like that config line at all
```
$line = 'art_search_limit="25"';
preg_match("/^;?([\w\d]+)\s+=\s+[\"]{1}(.*?)[\"]{1}$/", $line, $matches);
print "\n1";
print_r($matches);
preg_match("/^;?([\w\d]+)\s+=\s+[\']{1}(.*?)[\']{1}$/", $line, $matches);
print "\n2";
print_r($matches);
preg_match("/^;?([\w\d]+)\s+=\s+[\'\"]{0}(.*)[\'\"]{0}$/", $line, $matches);
print "\n3";
print_r($matches);
```

prints nothing
```
1Array
(
)

2Array
(
)

3Array
(
)
```

so it gets replaced with the default. that's going to be an important one to fix before we push everyone up again

**Comment by lachlan-00** [comment]
config items without a space.... will fix and update the dist for it.

**Comment by lachlan-00** [comment]
luckily it's only 2 items relating to art, fixed with c3513c74e and updating config lines that are 'wrong' will restore a 'foo = bar' line

![image](https://user-images.githubusercontent.com/1305249/129280900-596360f8-8a04-44f0-ab4f-0e1765fd02e6.png)
